src/main/java/org/cactoos/text/Replaced.java

```diff
@@ -82,7 +82,7 @@ public Replaced(
     public Replaced(
         final Text text,
         final Scalar<Pattern> regex,
-        final Func<Matcher, String> func
+        final Func<? super Matcher, String> func
```

Comment:
@DmitryBarskov Is it supposed to be `String` here?

Reply:
@andreoss it's possible to generalize it more by changing this type to `Func<? super Matcher, ? extends CharSequence> func`, but in this case I will have to change this line https://github.com/yegor256/cactoos/blob/cc5daf0ef6cecb699f6d6fb13695713010cba323/src/main/java/org/cactoos/text/Replaced.java#L95

to

```java
  func.apply(matcher).toString()
```

which looks a bit weird (just like we can pass `Func<? super Matcher, ?>` as `func` here).
WDYT?

Reply:
@DmitryBarskov `toString()` is fine if called on `CharSequence`